.github/workflows/ci.yml

```diff
@@ -80,7 +80,6 @@ jobs:
         with:
           token: ${{ secrets.CODECOV_TOKEN }} # not required for public repos
           files: lcov.info
-          fail_ci_if_error: true
```

Comment:
we could miss some important information without coverage and will end up with rerunning the job

we can try rerun it automatically like it's done [here](https://github.com/tagatac/bagoup/pull/57/files)